hasura/metadata/databases/default/tables/public_pending_app_metadata.yaml

```diff
@@ -0,0 +1,103 @@
+table:
+  name: pending_app_metadata
+  schema: public
+object_relationships:
+  - name: pending_app_metadata_team
+    using:
+      foreign_key_constraint_on: team_id
+insert_permissions:
+  - role: user
+    permission:
+      check: {}
+      columns:
+        - id
```

Comment:
@andy-t-wang As far as I know we shouldn't give permission to user to edit `id`, `created_at`, `updated_at` fields, it will be created automatically. We should only allow to insert fields that the user role will send directly from the client side. 
Same with update